apps-rendering/package.json

```diff
@@ -110,5 +110,8 @@
 		"whatwg-fetch": "3.6.19",
 		"winston": "3.11.0",
 		"winston-daily-rotate-file": "4.7.1"
+	},
+	"devDependencies": {
+		"@storybook/addon-webpack5-compiler-babel": "^3.0.3"
```

Comment:
Should this stay in the main `dependencies`?

Reply:
yes, good spot! storybook's auto-updater puts it here. I'll move it!